### DISCUSSION TITLE
Alerting Management

### DISCUSSION THREAD

**Discussion by tusharrcrm** [apache/skywalking#12807] [Q&A] [answered]
Since I couldnt find a system where we can manage our alarms, alerts from Skywalk UI. If there is any someone can let me know. 

I am using docker to run UI and oap containers. Each time to create a new alert I have to make changes in .yaml file and restart containers. Isnt there any more efficient approach, since restarting can lead to data loss for that particular time period.

**Comment by wu-sheng** [comment, marked as answer]
We don't provide management. You could update them on demand 

https://skywalking.apache.org/docs/main/next/en/setup/backend/backend-alarm/#update-the-settings-dynamically
